openedx/core/djangoapps/schedules/tests/test_utils.py

```diff
@@ -69,7 +72,16 @@ def test_safe_without_schedule(self):
         self.create_schedule()
         self.schedule.delete()
 
-        reset_self_paced_schedule(self.user, self.course.id, use_availability_date=False)
-        reset_self_paced_schedule(self.user, self.course.id, use_availability_date=True)
+        reset_self_paced_schedule(self.user, self.course.id, use_enrollment_date=False)
+        reset_self_paced_schedule(self.user, self.course.id, use_enrollment_date=True)
 
         assert Schedule.objects.count() == 0
+
+    @ddt.data(True, False)
+    def test_reset_before_course_starts(self, use_enrollment_date):
+        self.create_schedule(enrollment_offset=-5, course_start_offset=5)  # starts in 5 days, enrollment is now
```

Comment:
Enrollment is now feels weird with `enrollment_offest=-5`. Would have thought enrollment was 5 days ago. /shrug

Reply:
I agree that it looks weird. But the other tests were all written assuming that `enrollment_offset` would be +/- the course start date... it felt like too big a refactor to avoid the ugliness.